modules/tinymce/src/core/test/ts/browser/textpatterns/TextPatternsTest.ts

```diff
@@ -203,46 +199,8 @@ describe('browser.tinymce.plugins.textpattern.TextPatternPluginTest', () => {
     TinyAssertions.assertContentPresence(editor, { ol: 1, li: 2 });
   });
 
-  it('getPatterns/setPatterns', () => {
-    const editor = hook.editor();
-    // Store the original patterns
-    const origPatterns = editor.plugins.textpattern.getPatterns();
-
-    editor.plugins.textpattern.setPatterns([
-      { start: '#', format: 'h1' },
-      { start: '##', format: 'h2' },
-      { start: '###', format: 'h3' }
-    ]);
-
-    assert.deepEqual(
-      editor.plugins.textpattern.getPatterns(),
-      [
-        {
-          format: 'h3',
-          start: '###'
-        },
-        {
-          format: 'h2',
-          start: '##'
-        },
-
-        {
-          format: 'h1',
-          start: '#'
-        }
-      ],
-      'should be the same'
-    );
-
-    // Restore the original patterns
-    editor.plugins.textpattern.setPatterns(origPatterns);
-  });
-
-  // TODO TINY-3258 reenable this test when issues with Chrome 72-75 are sorted out
-  it('test inline and block at the same time', function () {
-    if (detection.browser.isChromium()) {
-      this.skip();
-    }
+  // TODO TINY-3258 trying to re-enable this...
```

Comment:
```suggestion
  // TODO TINY-3258 trying to re-enable this...
```
re-enabled this, it passed, so removing the comment and closing the Jira

Reply:
Looks like you didn't remove the comment here?

Reply:
Ahh, thanks